pom.xml

```diff
@@ -155,6 +155,7 @@
                 <version>${maven.compiler.version}</version>
                 <configuration>
                     <release>11</release>
+                    <encoding>UTF-8</encoding>
```

Comment:
Even though this fixes the errors that Jenkins was reporting, Jenkins is still showing warnings about the project using a platform-specific encoding for copying resources. Would it be better to put this configuration in the "project.build.sourceEncoding" property instead, so that it affects the project as a whole? The various Cynergy POMs are currently using that property to specify the UTF-8 encoding.